docs/Earn/liquid-staking/ksm/ksm-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Polkadot Staking Mechanics
+id: ksm-liquid-staking-mechanics
+---
+# Kusama Staking Mechanics
+
+:::note
+KSM Liquid Staking is almost identical to DOT Liquid Staking.
+:::
+
+KSM Liquid Staking lets the user stake their funds through the ANKR deposit address on the Polkadot network, accumulate rewards, and receive their stake+rewards when unstaking.
+
+ANKR principles for the staking are:
+
+* Receive KSM from the user to the ANKR deposit address.
+
+* Mint aKSMb on Ethereum. Make all claiming aKSMb transactions on the Ethereum network.
+
+* Use the fixed exchange rate 1:1 for aKSMb to KSM. aKSMb always represents 1 KSM token even though we sacrifice ERC-20 compatibility for this.
+
+* aKSMb ratio changes, thus all aKSMb balances “automatically” grow.
+
+Staking is well-described by its requirement, fees, rewards, validators, as well as smart contracts and workflow that shows the API endpoints to call to interact with these smart contracts.
```

Comment:
Please rephrase as previously.

Reply:
Also, how do users interact with the API endpoints?

Reply:
nohow. I've removed this part, we don't expose the API for Kusama/Polkadot.